### ISSUE TITLE
Provide a backtest example

### ISSUE THREAD

**Issue by Josema** [Mathieu2301/TradingView-API#207]
Hi, can you provide a backtest example?

**Comment by tommiehansen** [comment]
Yes, i'd also like to see this. Could possible compile some sort of example by simple reading all available code in the repo, all issues and all libraries dependent on this library but ... yeah... :)
